index.js

```diff
@@ -26,6 +26,8 @@ const ServerResponse = require('fwsp-server-response');
 let serverResponse = new ServerResponse();
 const UMFMessage = require('fwsp-umf-message');
 
+const RedisConnection = require('./redis');
```

Comment:
I'm thinking it might make sense to make a fwsp-redis-connection module so we can use this from anywhere we're connecting to redis (e.g. the cacher module, and some services which use redis directly).